py-polars/polars/utils/convert.py

```diff
@@ -154,27 +149,28 @@ def _to_python_time(value: int) -> time:
         minutes, seconds = divmod(seconds, 60)
         hours, minutes = divmod(minutes, 60)
         return time(
-            hour=hours, minute=minutes, second=seconds, microsecond=nanoseconds // 1000
+            hour=hours, minute=minutes, second=seconds, microsecond=nanoseconds // 1_000
         )
 
 
-def _to_python_timedelta(value: int | float, time_unit: TimeUnit = "ns") -> timedelta:
+def _to_python_timedelta(
+    value: int | float, time_unit: TimeUnit | None = "ns"
+) -> timedelta:
     if time_unit == "ns":
-        return timedelta(microseconds=value // 1e3)
+        return timedelta(microseconds=value // 1_000)
     elif time_unit == "us":
         return timedelta(microseconds=value)
     elif time_unit == "ms":
         return timedelta(milliseconds=value)
-    else:
-        raise ValueError(
-            f"`time_unit` must be one of {{'ns', 'us', 'ms'}}, got {time_unit!r}"
-        )
+    raise ValueError(
+        f"`time_unit` must be one of {{'ms', 'us', 'ns'}}, got {time_unit!r}"
+    )
 
 
 @lru_cache(256)
 def _to_python_date(value: int | float) -> date:
     """Convert polars int64 timestamp to Python date."""
-    return (EPOCH_UTC + timedelta(seconds=value * 86400)).date()
+    return (EPOCH_UTC + timedelta(seconds=value * SECONDS_PER_DAY)).date()
```

Comment:
Maybe we can set `SECONDS_PER_DAY` to `86_400` instead of doing the multiplication.